webapp/src/components/ProgramCalendar.tsx

```diff
@@ -69,6 +69,26 @@ const ProgramCalendar = ({ program }: ProgramCalendarProps) => {
         minHeight: '4%',
       };
     }
+    if (event.allDay && currentView !== 'agenda') {
+      style = {
+        backgroundColor: 'midnightblue',
+      };
+    }
+
+    if (event.allDay && currentView === 'week') {
+      style = {
+        height: '190%',
+        backgroundColor: 'midnightblue',
+      };
+    }
+
+    if (event.allDay && currentView === 'day') {
+      style = {
+        height: '220%',
+        backgroundColor: 'midnightblue',
+      };
+    }
+
```

Comment:
Great job, Sanaz! Everything works in my local machine. 
As for the height of the events, I prefer to leave them at their default height as Wendy commented before.

Reply:
Ok. Thanks for your feedback.